webapp/greenhouse_api.py

```diff
@@ -22,6 +22,7 @@ def get_vacancies(department):
     vacancies = []
     for job in feed["jobs"]:
         feed_department = remove_hyphens(job["metadata"][2]["value"])
```

Comment:
I'm not a fan of variable re-assignment. I'd prefer something like...
```suggestion
        feed_department = parse_feed_department(
            remove_hyphens(job["metadata"][2]["value"]))
```

Reply:
Done